e2e-tests/monitoring-2-0/run

```diff
@@ -87,6 +106,27 @@ get_metric_values haproxy_backend_active_servers $namespace-$cluster-haproxy-0 a
 
 desc 'check QAN data'
 get_qan20_values $cluster-pxc-0 admin:admin
+
+nodeList=($(get_node_id_from_pmm))
+nodeList_from_pmm=($(is_node_id_exists "${nodeList[@]}"))
+for node_id in "${nodeList_from_pmm[@]}"; do
+	if [ -z "$node_id" ]; then
+		echo "Can't get $node_id node_id from PMM server"
+		exit 1
+	fi
+done
+
+kubectl_bin patch pxc ${cluster} --type json -p='[{"op":"add","path":"/spec/pause","value":true}]'
+wait_for_delete "pod/${cluster}-pxc-0"
+
+is_nodes_id_exist_in_pmm=($(is_node_id_exists "${nodeList[@]}"))
```

Comment:
`node_id_exists_pmm` or if we want a longer version `does_node_id_exists_in_pmm`

Reply:
https://github.com/percona/percona-xtradb-cluster-operator/pull/1416/commits/b19ff569e811763cde0a5ba8a2c67fe6dac8ffde done